paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -168,6 +170,29 @@ internal class DefaultFlowController @Inject internal constructor(
         configuration: PaymentSheet.Configuration?,
         callback: PaymentSheet.FlowController.ConfigCallback
     ) {
+        try {
+            configuration?.validate()
+            if (clientSecret.value.isBlank()) {
```

Comment:
same here as my comment earlier, it seems like we could consolidate the validation code into one central place rather than having ~similar validation code spread out across multiple entry points

Reply:
same as above.  Ultimately I think we need to look at how we can reuse the DefaultFlowController in PaymentSheet single step, but that is a much bigger lift, if even possible.